tests/settings.py

```diff
@@ -26,7 +26,7 @@
     # test project
     'test_project',
     # admin
-    'django.contrib.admin',
+    'openwisp_utils.admin_theme.apps.OpenwispAdminConfig',
```

Comment:
this change should not be needed, also because `openwisp_utils.admin_theme` is already called earlier:
https://github.com/openwisp/openwisp-utils/blob/9f065e3d3be5c84f115cd3b4f9744c5e55fd7feb/tests/settings.py#L17